recipes/desdeo-vis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "desdeo-vis" %}
+{% set version = "0.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d70a42e3aa6e4f8ef1c731b17a7baefea39f9d6d015597827fad9b9e42f27e30" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - python >= 3.6
+    - ipywidgets >=7.0.0
+    - widgetsnbextension
+  run:
+    - python >= 3.6
+    - ipywidgets >=7.0.0
+    - widgetsnbextension
+    - desdeo >=0.1.3
+    - matplotlib
+    - altair
+    - vega
+
+test:
+  imports:
+    - desdeo_vis
+    - desdeo_vis.nbextension
+    - desdeo_vis.plot
+    - desdeo_vis.widget
+
+about:
+  home: https://github.com/industrial-optimization-group/desdeo-vis
+  license: MPL-2.0
+  license_file: 'LICENSE'
```

Comment:
Despite it passing here, apparently the license file cannot be found in the feedstock build.

ref: https://circleci.com/gh/conda-forge/desdeo-vis-feedstock/2